### DISCUSSION TITLE
Integration dans google Sheet

### DISCUSSION THREAD

**Discussion by Galpamort** [bokub/conso-api#71] [Aide]
Bonjour,

Est-il possible d'intégrer cela dans google sheet (via app script) afin de récupérer les données ?
Avez-vous un code à proposer ?

Merci

**Comment by bokub** [comment]
Salut 

Tu peux utiliser mon module [bokub/linky](https://github.com/bokub/linky) pour exporter tes données au format CSV

**Comment by bokub** [comment]
Je ne suis pas certain d'avoir répondu à ta question mais je n'ai pas la réponse concernant app script.

**Reply by Galpamort** [reply]
C'est ce que souhaiterai faire, mais c'est l'intégration dans google sheet qui me pose problème ^^

**Reply by LoicChabut** [reply]
@Galpamort je pense que c'est possible. J'ai déjà utilisé Google Apps Script pour récupérer des données de Stripe et Pennylane via leurs API respectives.
